tests/validation.rs

```diff
@@ -150,17 +156,14 @@ fn bytes_fee_cant_overflow() {
         .change_output(AssetId::default())
         .build();
 
-    let mut interpreter = Interpreter::with_memory_storage();
-    let result = interpreter.transact(transaction);
-    assert!(matches!(
-        result,
-        Err(InterpreterError::ValidationError(ValidationError::ArithmeticOverflow))
-    ));
+    Interpreter::with_memory_storage()
+        .transact(transaction)
+        .expect("gas factor won't allow overflow");
```

Comment:
is this true if someone sets gas factor to 1?

Reply:
Good catch, updated